### PR TITLE
Api 12146 json cleanup

### PR DESCRIPTION
**Pull request by JamesDanielBell-VA**
https://vajira.max.gov/browse/API-12146
-moving v1 fields style to camelCase from snake_case
-this PR also contains API-12142

### CODE REVIEW

**Comment by mmccollum-va** [comment]
The following files should be removed as a result of the efforts to deprecated `geo+json` in [API-12142](https://vajira.max.gov/browse/API-12142):

- `facilities`
    - `test/resources/v1/`
        - `read-benefits-geojson.json`
        - `read-cemetery-geojson.json`
        - `read-health-geojson.json`
        - `read-state-cemetery-geojson.json`
        - `read-vet-center-geojson.json`
        - `search-bbox-geojson.json`
        - `search-ids-geojson.json`
        - `search-lat-long-geojson.json`
        - `search-state-geojson.json`
        - `search-zip-geojson.json`

**Comment by JamesDanielBell-VA** [comment]
thank you for the catch!
Will update PR

**Review by mmccollum-va — APPROVED** [review]

**Comment by JamesDanielBell-VA** [comment]
Thanks for the catch on minTime and maxTime. The testerino was indeed for testing purposes. It was removed this morning following Adam's feedback in standup

**Review by ChrisBratti-VA — APPROVED** [review]

**Review by mmccollum-va — APPROVED** [review]

**Review by giangnguyen2 — APPROVED** [review]

File: facilities-api/src/test/resources/v1/search-zip-geojson.json [hunk 1]
```diff
@@ -13,42 +13,41 @@
       "properties" : {
         "id" : "vha_675GA",
         "name" : "Viera VA Clinic",
-        "facility_type" : "va_health_facility",
+        "facilityType" : "va_health_facility",
```
Comment (as-va) [inline facilities-api/src/test/resources/v1/search-zip-geojson.json:16]: GeoJson is deprecated in V1 - this file should be removed.

File: facilities-api/src/test/resources/v1/read-health-geojson.json [hunk 2]
```diff
@@ -10,64 +10,63 @@
   "properties" : {
     "id" : "vha_402GA",
     "name" : "Caribou VA Clinic",
-    "facility_type" : "va_health_facility",
+    "facilityType" : "va_health_facility",
```
Comment (as-va) [inline facilities-api/src/test/resources/v1/read-health-geojson.json:13]: GeoJson is deprecated in V1 - this file should be removed.

**Review by as-va — CHANGES_REQUESTED** [review]
Everything looks good except for the two GeoJson resource files in the V1 package. Remove those and we'll merge in the rest.

**Review by as-va — APPROVED** [review]

**Pull request merged**